include/crm/common/util_compat.h

```diff
@@ -69,6 +69,9 @@ char *pcmk_format_nvpair(const char *name, const char *value,
 //! \deprecated Use a standard printf()-style function instead
 char *pcmk_format_named_time(const char *name, time_t epoch_time);
 
+//! \deprecated Use g_str_hash() instead
+#define crm_str_hash g_str_hash_traditional
+
 //! \deprecated Use g_hash_table_new_full() instead
 static inline GHashTable *
 crm_str_table_new(void)
```

Comment:
I am really liking having all the deorecated code in its own headers.  It makes it a lot more obvious what we shouldn't be using and what can go away in the future.  The only other thing that might be nice to do would be include a comment on when things are expected to be removed.